receivers/slack/config_test.go

```diff
@@ -256,7 +255,7 @@ func TestNewConfig(t *testing.T) {
 			settings: FullValidConfigForTesting,
 			expectedConfig: Config{
 				EndpointURL:    "http://localhost/endpoint_url",
-				URL:            "http://localhost/url",
+				URL:            "http://localhost/endpoint_url",
```

Comment:
This was changed to make the tests pass, but I think these tests need to be changed as they are based on the assumption you can configure both incoming webhooks and bot tokens at the same time.